.github/workflows/deploy.yaml

```diff
@@ -5,18 +5,20 @@ on:
   push:
     branches:
       - main
+      - dev
```

Comment:
I am missing a big chunk of logic for dev deployment which should include:
- set appropriate repo
- set appropriate channel